Formula/rex.rb

```diff
@@ -206,9 +214,16 @@ class Rex < Formula
   end
 
   def install
+    # First prevent trying to sniff around elsewhere for required modules.
+    ENV.delete("PERL5LIB")
+
     ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
     ENV.prepend_path "PERL5LIB", libexec/"lib"
 
+    # If you're using a new enough perl it'll blow up & require your
+    # confirmation that you know what you're doing. Skip that.
+    ENV["PERL_CANARY_STABILITY_NOPROMPT"] = "1"
```

Comment:
Eh. There's a new Perl, upstream doesn't recognise it as stable yet & flags it as you're using a beta build I hope you know what the heck you're doing, etc. I can find someone to report it to but upstream are gonna basically be like "Well, yeah".